docs/02-Install.md

```diff
@@ -1,13 +1,140 @@
 # 2.0 - Install
 
-## 2.1 Dependencies
+## 2.1 Dependencies and installation
 
-## 2.2 Setup
+Dependencies and installation information are documented on the LORIS-MRI
+  [README.md](../README.md) file.
 
-## 2.3 Configuration
 
-## 2.4 Pipeline flow
+## 2.2 Configuration
 
-## 2.5 Common errors
+Following a successful install, some configurations and customizations are needed
+on the database, LORIS and LORIS-MRI sides as outlines in the following sub-sections.
 
-[FAQ](AppendixA-FAQ.md)
+### 2.2.1 Database
+
+1. `psc` table
+2. `Visit_Windows` table
+3. `mri_scan_types` table
+4. `mri_protocol` table
+5. `Config` table
+
+The items in 5. cna also be accessed in the LORIS front-end, under the Admin menu, 
+the `Config` module. Under the section `Imaging Pipeline`, verify/set the following 
+config settings:
+ * `Loris-MRI Data Directory`
+ * `Study Name`
+ * `User to notify when executing the pipeline`
+ * `Full path to get_dicom_info.pl script`
+ * `Path to Tarchives`
+
+Under the section `Paths`
+ * `imagePath`
+
+**NEED TO TALK ABOUT THE CREATION OF NIFTI OPTIONS AS IT IS REFERENCED IN THE
+  PIPELINE FLOW DOCUMENTATION.**
+
+### 2.2.2 LORIS
+
+
+### 2.2.3 LORIS-MRI 
```

Comment:
add a note here about the environment file and specifically, the tmp_dir where all temporary processing happens in case projects want to customize this (default is /tmp)